src/components/patients/patientTable/index.tsx

```diff
@@ -17,20 +17,23 @@ const PatientTable = () => {
   const [patients, setPatients] = React.useState([] as Patient[]);
 
   const renderPatientAttribute = (
-    attribute: "firstName" | "lastName" | "age",
+    attribute: "id" | "firstName" | "lastName" | "age",
     index: number
   ) => (
     <Cell>
       <React.Fragment>
         <Link to={`${ROUTE_PATIENT}/${patients[index].id}`}>
-          {patients[index][attribute]}
+          {patients[index][attribute] || "unknown"}
         </Link>
       </React.Fragment>
     </Cell>
   );
 
   React.useEffect(() => {
-    setPatients(DATA_TEST);
+    let patientPromise = getPatients();
+    patientPromise.then((value: any) => {
```

Comment:
Can you use async/await in useEffect?
Something like
```
React.useEffect(async () => {
    const patients = await getPatients();
    setPatients(patients);
}, []);
```

Reply:
useEffect does not support async functions yet. but you can do:
```
React.useEffect(() => {
    const fetchPatients = async () => {
        const patients = await getPatients();
        setPatients(patients);
    }
    fetchPatients()
}, []);
```
However, it's fine to use .then() in React.useEffect since it does not handle async/await "natively"